Dockerfile.cli

```
@@ -0,0 +1,29 @@
+FROM node:10-alpine
+
+WORKDIR /gatsby
+
+ARG GATSBY_VERSION=^2.0.0
+ARG VIPS_VERSION=8.6.3-r0
+
+# Install dev dependencies
+RUN set -x \
+  && apk add --no-cache vips-dev=${VIPS_VERSION} \
+  --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing \
```

Running into permission errors with invalid SSL cert on this domain:

```
fetch https://dl-3.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz
SSL certificate subject doesn't match host dl-3.alpinelinux.org
WARNING: Ignoring https://dl-3.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz: Permission denied
```

Official github repo for sharp [advises to use the following repository domains instead](https://github.com/lovell/sharp/blob/master/docs/install.md#alpine-linux): 

```suggestion
    --repository https://alpine.global.ssl.fastly.net/alpine/edge/testing/ \
    --repository https://alpine.global.ssl.fastly.net/alpine/edge/main
```

---

So this is not needed anymore as we skip libvips. Thanks!